lib/tasks/end_users_switch.rake

```diff
@@ -0,0 +1,11 @@
+
+namespace :end_user_switch do
+  desc 'Disable end user switch for all accounts'
+  task disable_all: :environment do
+    Settings.find_in_batches(batch_size: 500) do |group|
+      ids = group.collect(&:id)
+      Settings.where(id: ids).update_all(end_users_switch: 'denied')
```

Comment:
good for me but you should check with ops that `bacth_size` of 500 with `update_all` and later `sleep(2)` is the way to go 😄